### PR TITLE
Clarification on TSE use

### PR DESCRIPTION
**Pull request by fiscoal**

### CODE REVIEW

**Review by TSchmiedlechner — CHANGES_REQUESTED** [review]
There's also a Swissbit TSE-as-a-Service available; hence, the limitations mentioned should be rephrased to generally only refer to "standalone TSEs"/"Einzelprodukt-TSEs". 

Outlet-based cloud TSEs-as-a-Service support all scenarios.

**Comment by fiscoal** [comment]
@TSchmiedlechner, this was already taken into account. Which paragraph would you be referring to?

File: i18n/de/middleware/README.md [hunk 1]
```diff
@@ -632,6 +632,13 @@ Im Kapitel [Template Beispiele](#template-beispiele) finden Sie ein Template, da
 Eine Schritt-für-Schritt Anleitung für diese Konfiguration kann [hier](step-by-step/3-hardware-tse-at-the-main-cash-register-for-several-additional-cash-registers/README.md) nachgelesen werden.
 
 ### Eine Cloud-TSE für mehrere Kassen
+
+:::caution
+
+Diese Setup Szenario erfordert eine **fiskaly TSE als Teil eines TSE-as-a-Service Pakets**. Weder eine Swissbit Cloud TSE, noch ein fiskaly TSE Einzelprodukt wäre in diesem Anwendungsfall unterstützt.
```
Comment (TSchmiedlechner) [inline i18n/de/middleware/README.md:638]: We could add here that if people want to use a Single Swissbit Cloud TSE with multiple Queues, they should use a the "1 SCU <-> n Queues" approach

File: middleware/README.md [hunk 2]
```diff
@@ -635,7 +635,14 @@ In the chapter [Template examples](#template-examples) you can find a template t
 A step-by-step guide for this configuration can be found [here](step-by-step/3-hardware-tse-at-the-main-cash-register-for-several-additional-cash-registers/README.md).
 
 ### A cloud TSE for multiple cash registers
-The fiskaltrust.Middleware runs on each cash register. The CashBox of each cash register configures the fiskaltrust.Middleware instance with its own Queue and its own SCU. Each SCU accesses the same cloud TSE. This scenario enables cloud TSEs to be saved. One advantage here is that the SCU does not become a bottleneck, since each POS has its own SCU. However, since all requests are sent to the same cloud TSE, the TSE becomes a bottleneck. Furthermore, both possible [performance bottlenecks](#performance recommendation) in the cloud TSE and our [fair use policy](https://github.com/fiskaltrust/productdescription-de-doc/blob/master/product-service-description/market-de-fair-use-policy.md) must also be taken into account here.
+
+:::caution
+
+This setup scenario requires a **fiskaly TSE as part of a TSE-as-a-Service package**. Neither a Swissbit Cloud TSE nor a fiskaltrust standalone fiskaly TSE would be supported for this use case.
```
Comment (TSchmiedlechner) [inline middleware/README.md:641]: Same as above

File: i18n/de/middleware/README.md (hunk 1 above)
Comment (fiscoal) [reply i18n/de/middleware/README.md:638]: True, though, that should also work with a standalone fiskaly SCU, right? In that case the whole sentence should be rephrased.

**Pull request merged**